Add RenameTestCases transformer (#191)
MarketSquare/robotframework-tidy: docs/source/transformers/RenameTestCases.rst

.. _RenameTestCases:

RenameTestCases
================================

Enforce test case naming.

> RenameTestCases is not included in default transformers that's why you need to call it with ``--transform`` explicitly::

Reviewer: ```suggestion
RenameTestCases is not included in default transformers, that's why you need to call it with ``--transform`` explicitly::
```

    robotidy --transform RenameTestCases src

Or configure `enable` parameter::

    robotidy --configure RenameTestCases:enabled=True

> This transformer capitalize first letter of test case name, remove trailing dot and strip leading/trailing whitespace.

Reviewer: ```suggestion
This transformer capitalizes first letter of test case name, removes trailing dot and strips leading/trailing whitespace.
```

It is also possible to configure `replace_pattern` parameter to find and replace regex pattern. Use `replace_to`

> to set replacement value. This configuration::

Reviewer: ```suggestion
to set a replacement value. This configuration::
```
